manager.go

```diff
@@ -222,52 +221,61 @@ func (m *Manager) GetStats() (Stats, error) {
 
 // GetRetries returns the set of retry jobs for the manager
 func (m *Manager) GetRetries(page uint64, pageSize int64, match string) (Retries, error) {
-	retries := Retries{}
+	// TODO: add back pagination and filtering
 
 	storeRetries, err := m.opts.store.GetAllRetries()
 	if err != nil {
-		return retries, err
+		return Retries{}, err
 	}
-	retryStats := m.opts.client.ZScan(m.opts.Namespace+storage.RetryKey, page, match, pageSize).Iterator()
 
-	var messages []*Msg
-
-	for retryStats.Next() {
-		msg, err := NewMsg(retryStats.Val())
-		if err != nil {
-			break
-		}
-		retryStats.Next()
-		messages = append(messages, msg)
+	retryJobStats, err := getRetryJSON(storeRetries.RetryJobs)
+	if err != nil {
+		return Retries{}, err
 	}
 
+	return Retries{
+		TotalRetryCount: storeRetries.TotalRetryCount,
+		RetryJobs:       retryJobStats,
+	}, nil
+}
+
+func getRetryJSON(retryStats []string) ([]RetryJobStats, error) {
 	var retryJobStats []RetryJobStats
+	for i := 0; i < len(retryStats); i++ {
```

Comment:
nit: could use `for i := range retryStats` here

Reply:
good suggestion! let me fix this ...